app/src/main/java/com/example/nigeriatelemedicineapp/registerpatient/RegisterPatientActivity.kt

```diff
@@ -45,7 +47,7 @@ class RegisterPatientActivity : AppCompatActivity() {
     fun setUpUI() {
         setSupportActionBar(binding.toolbar)
         val actionBar = supportActionBar
-        actionBar!!.title = getString(R.string.appointmentFormheading)
+        actionBar!!.title = "consult a doctor"
```

Comment:
Let's go ahead and change the name of the "Register Patient" button on the homescreen to match the title of this activity. Also let's do title case capitalization.

Reply:
Yup, that's a really nice idea.

Reply:
Done